lib/nexthop_group.h

```diff
@@ -64,7 +64,7 @@ void copy_nexthops(struct nexthop **tnh, struct nexthop *nh,
 
 struct nexthop_hold {
 	char *nhvrf_name;
-	union sockunion addr;
+	union sockunion *addr;
```

Comment:
Shouldn't this be it's own commit?  The change from a addr to *addr?

Reply:
I don't see the point of it. The first commit is pretty small already, and the change from `addr` to `addr *` only makes sense when the nexthop address becomes optional, which is what the first commit does.

Reply:
So the point is that I want to be able to just see the meat of the important changes in one spot.  As it is I'm having to wade through trivial changes that are preparatory in nature to make sure that the commit is right.  Which makes it harder for me to review and understand what is going on.

Reply:
Ok, I'll create a separate commit for that.